src/site/content/en/blog/lighthouse-whats-new-6.0/index.md

```diff
@@ -0,0 +1,463 @@
+---
+title: "What's New in Lighthouse 6.0"
+authors:
+  - cjamcl
+date: 2020-05-18
+hero: lighthouse-logo.svg
```

Comment:
lol responsive svg's:

![image](https://user-images.githubusercontent.com/39191/82106984-32a58780-96d9-11ea-9ee9-c00623ecf563.png)

Reply:
good catch. https://github.com/GoogleChrome/web.dev/issues/2892

Reply:
was there something to do for this? logo looks kinda big :)

Reply:
I don't think so. The bug i opened was to not do srcsetting on svg but I don't think it's affecting how it's being displayed. Might try a different image if  you want it to be smaller?